Doc/c-api/long.rst

```diff
@@ -732,7 +732,8 @@ The :c:type:`PyLongWriter` API can be used to import an integer.
 
    *negative* is ``1`` if the number is negative, or ``0`` otherwise.
 
-   *ndigits* is the number of digits in the *digits* array.
+   *ndigits* is the number of digits in the *digits* array. It must be
+   positive.
```

Comment:
To avoid any ambiguities, I would say "nonnegative" or "greater than or equal to 0".

Reply:
Ok, done